sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -87,6 +88,12 @@ private void processNonCachedEvent(SentryEvent event) {
       event.setSdk(options.getSdkVersion());
     }
 
+    for (Map.Entry<String, String> tag : options.getTags().entrySet()) {
+      if (event.getTag(tag.getKey()) == null) {
```

Comment:
what's about `containsKey` instead?

Reply:
This would be then:
```java
if (event.getTags() == null || !event.getTags().containsKey(tag.getKey())) {
    event.setTag(tag.getKey(), tag.getValue());
}
```
I think I prefer the original version.

Reply:
true, `event.getTags()` can be null, my bad